packages/edge-login-ui-rn/src/components/screens/existingAccout/ChangeRecoveryScreen.js

```diff
@@ -100,6 +100,10 @@ class ChangeRecoveryScreenComponent extends React.Component<Props, State> {
       this.state.question2 === s.strings.choose_recovery_question ||
       this.state.question1 === this.state.question2
 
+    if (this.state.question1 === this.state.question2) {
+      showError(s.strings.change_recovery_same_question)
```

Comment:
Don't you need to return here? 
Is showError enough so the function won't continue?

Reply:
That should be correct cause any errors beside that will indicated